x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -421,6 +422,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
                 from: '2016-02-07T00:00:00.000Z',
                 to: '2016-02-11T23:59:54.000Z',
               },
+              query: { query: '', language: SEARCH_QUERY_LANGUAGE.KUERY },
```

Comment:
There is no need in storing an empty string. I think the source code should omit the `query` altogether in that case and the assertion of the test should remain the same.

Reply:
Updated here [`25da066` (#140147)](https://github.com/elastic/kibana/pull/140147/commits/25da06687b0ccb2f28ff0a117ef5eb53bbed6a4e)